src/main/java/org/asamk/signal/commands/UpdateGroupCommand.java

```diff
@@ -168,7 +177,13 @@ public void handleCommand(final Namespace ns, final Signal signal) throws Comman
         try {
             var newGroupId = signal.updateGroup(groupId, groupName, groupMembers, groupAvatar);
             if (groupId.length != newGroupId.length) {
-                writer.println("Created new group: \"{}\"", Base64.getEncoder().encodeToString(newGroupId));
+                String encodedGroup = Base64.getEncoder().encodeToString(newGroupId);
+                if (ns.get("output") == OutputType.JSON) {
```

Comment:
There's a second `handleCommand` method above, please also add the json output there.